### ISSUE TITLE
typescript类型提示问题

### ISSUE THREAD

**Issue by zero9527** [microsoft/vscode#233150]
Type: <b>Bug</b>

使用一段时间后，鼠标移到变量或者typescript类型上面，会有以下问题：
- 没有类型提示，没有任何反应，此时按住command同时点击鼠标左键可以正常跳到实现

正常会有浮层提示类型，重启VS Code之后正常，但是使用一段时间之后还会出现

VS Code version: Code 1.95.1 (65edc4939843c90c34d61f4ce11704f09d3e5cb6, 2024-10-31T05:14:54.222Z)
OS version: Darwin x64 22.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|4, 3, 2|
|Memory (System)|16.00GB (1.19GB free)|
|Process Argv|--crash-reporter-id df018b86-dcb3-4093-a0fe-1df1f758d245|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
markdown-mermaid|bie|1.26.0
vscode-eslint|dba|3.0.13
prettier-vscode|esb|11.0.0
githd|hui|2.5.3
vscode-language-pack-zh-hans|MS-|1.95.2024103009
volar|Vue|2.1.10
material-theme|zhu|3.17.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
vscrpc:30673769
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
jg8ic977:31013176
dvdeprecation:31068756
dwnewjupytercf:31046870
impr_priority:31102340
nativerepl1:31139838
refactort:31108082
pythonrstrctxt:31112756
cf971741:31144450
iacca1:31171482
notype1cf:31157160
5fd0e150:31155592
dwcopilot:31170013

```

</details>

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by zero9527** [comment]
> Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

No, i did not install the latest VS Code insiders build, but i will give it a try.

I have tried Troubleshoot issue and the problem, but it only worked after restarting VSCode
![Image](https://github.com/user-attachments/assets/65800157-ce1f-450d-9856-5c20e2b44d3e)

**Comment by zero9527** [comment]
> Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

Thanks for reply.

This problem may be caused by the extension `Git History Diff`, and should not be a problem with VSCode itself.

**Comment by mjbvz** [comment]
Please share the code that demonstrates the issue as text so we can easily test this. Thanks

**Comment by zero9527** [comment]
> Please share the code that demonstrates the issue as text so we can easily test this. Thanks

In `age: '2'`, when you move the mouse over it, an error message is expected to appear. 
It will generally disappear after a few minutes. 
It is expected that the message will still appear normally after multiple edits.

```ts
type foo = { name: string; age: string };
type coo = { age: number; sex: string };

type Merge<T extends object, U extends object> = U & {
  [K in keyof T as K extends keyof U ? never : K]: T[K];
};

type Result2 = Merge<foo, coo>; // expected to be {name: string, age: number, sex: string}

const res: Result2 = {
  name: '',
  age: '2',
  sex: ''
};
```
